### PR TITLE
UI text labels localization bug

### LINKED ISSUE #5015: [bug]: UI text labels not displaying localization or not available for translation
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### VRAM

8GB

### What version did you experience this issue on?

3.3.0

### What happened?

Several text labels in the UI are not converted to the related localized version, or are not available on Weblate for translation. I am reporting this relatively to the Italian localization, but this seems to happen also with other localizations.

I took screenshots of the UI highlighting the labels that don’t show up correctly for one of the two above reasons.

Compiled InvokeAI with “main” branch and used the latest it.json file available on Weblate.

MAIN UI - SETTINGS

immagine

All the above three strings are not available on Weblate.

MAIN UI – IMAGE VIEWER

immagine
 
The above is not available on Weblate.

MAIN UI – IMAGE ACTIONS MENU

immagine

“Change Board” is not displaying the translation, while “Unstar Image” isn’t available on Weblate.

 

“ESRGAN Model” isn’t available on Weblate.

immagine

MAIN UI – BOARDS

immagine
 
“Delete Board” is missing from Weblate.

immagine
 
“Delete” doesn’t show the localized text.
“Deleted boards cannot be restored.”, “Delete Board Only”, and “Delete Board and Images”  are not available on Weblate.

MAIN UI - GENERATION PARAMETERS

immagine
 
“Drop” isn’t available on Weblate.

immagine 

“Add LoRA” isn’t available on Weblate.

immagine

“Prompts Preview” isn’t available on Weblate.

immagine
 
The entire “High Resolution Fix” panel is missing on Weblate.

IMAGE TO IMAGE

immagine
 
“Initial Image”, “No initial image selected” are not using the localized text.
The labels of the two buttons on the top right corner, “Upload Initial Image” and “Reset Initial Image”, are not available on Weblate.

UNIFIED CANVAS

 
immagine

Both “Mode” and “Text to Image” are not using the localized texts.

immagine

“Unmasked”, “Mask” and “Mask Edge” are not available on Weblate.

WORKFLOW EDITOR

immagine

Not sure if the key settings.general is being reused. In the affirmative, then it doesn’t display the localized text. The same for “Advanced” and the common.advanced key. If these are instead different keys, then these two labels are missing on Weblate.

 
immagine

“Linear”, “Details”, “Outputs”, “Data” and “Template” aren’t available on Weblate.

MODELS MANAGER

immagine

All the highlighted labels are not displaying the localized text. Not sure about “No Model Selected” because the only key is related to the generation parameters and not the Model Manager.

IMPORT MODELS

immagine

“Simple” is missing from Weblate, “Advanced” doesn’t show the localized text.

MERGE MODELS

immagine

“Model Type” doesn’t display the localized text.

MODELS SETTINGS

immagine

“Sync Models” isn’t displaying the localized text.

QUEUE MANAGEMENT

 
immagine

“STATUS” and “BATCH” don’t display the localized text.

“TIME” and “BATCH FIELS VALUES” aren’t available on Weblate.

### Screenshots

_No response_

### Additional context

_No response_

### Contact Details

_No response_

### PR DESCRIPTION
**Pull request by rohinish404**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [X] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Community Node Submission

## Have you discussed this change with the InvokeAI team?
- [X] Yes
- [ ] No, because:

      
## Have you updated all relevant documentation?
- [ ] Yes
- [ ] No

## Description
This pr is still under development.

## Related Tickets & Documents

- Related Issue #5015
- Closes #5015

## Added/updated tests?

- [ ] Yes
- [X] No : _please replace this line with details on why tests
      have not been included_

### CODE REVIEW

**Comment by rohinish404** [comment]
i tried my best to change all the strings mentioned in the issue still if there's some left please let me know! :)

**Comment by hipsterusername** [comment]
Thanks for your help!

**Comment by rohinish404** [comment]
> Thanks for your help!

Happy to help! :)

**Comment by rohinish404** [comment]
Making this pr again draft as new bugs have been found!

File: invokeai/frontend/web/src/features/modelManager/subpanels/AddModelsPanel/AdvancedAddModels.tsx
Comment (psychedelicious) [inline invokeai/frontend/web/src/features/modelManager/subpanels/AddModelsPanel/AdvancedAddModels.tsx:1]: This needs to use the `useTranslation()` hook

**Review by psychedelicious — CHANGES_REQUESTED** [review]
Requested one change.

Once that issue is fixed, let's get this merged, despite there being some untranslated strings in the app.

Then, I'll PR a change that will let us more effectively address all missing translations.

File: invokeai/frontend/web/src/features/modelManager/subpanels/AddModelsPanel/AdvancedAddModels.tsx
Comment (rohinish404) [reply invokeai/frontend/web/src/features/modelManager/subpanels/AddModelsPanel/AdvancedAddModels.tsx:1]: resolved with latest commit

**Review by psychedelicious — APPROVED** [review]
Thanks @rohinish404!

**Pull request merged**